pylint/testutils.py

```diff
@@ -256,7 +256,6 @@ def walk(self, node):
 # Init
 test_reporter = TestReporter()
 linter = PyLinter(pylint.config.Configuration())
-linter.set_reporter(test_reporter)
```

Comment:
Because linters don't handle reprts now, this was breaking the setup for _all_ tests. I deleted it so I could run my tests, but I didn't check the impact on other tests as many tests are failing at the moment.